mosviz/loaders/test/sample_test.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import
+
+def test_sample():
+    assert(1==1)
```

Comment:
The travis build reported a failure if there wasn't an actual test running.  i.e. 0 tests != passing tests.  So i put one in there just so travis wouldn't show failures.  I'll make it a little more realistic, so we're not lying to ourselves - even if it just tests an import or something